ui/src/app/components/runtime-configuration-panel/customize-panel.tsx

```diff
@@ -284,15 +286,22 @@ export const CustomizePanel = ({
       </div>
       <DiskSelector
         diskConfig={analysisConfig.diskConfig}
-        onChange={(diskConfig) =>
+        onChange={(change: Partial<DiskConfig>) => {
+          const diskConfig = maybeWithExistingDiskName(
```

Comment:
all the `onChange`s downstream of this were doing this transformation, so let's just define it here